weldx/core.py

```diff
@@ -270,8 +270,9 @@ class TimeSeries:
     def __init__(
         self,
         data: Union[pint.Quantity, MathematicalExpression],
-        time: Union[None, pd.TimedeltaIndex, pint.Quantity] = None,
+        time: Union[types_time_like, Time] = None,
         interpolation: str = None,
+        reference_time: pd.Timestamp = None,
```

Comment:
Also not related to this PR, but a reminder to find a common interface name for the reference time and apply it to all functions